web/src/features/hfiCalculator/components/prepLevel.ts

```diff
@@ -1,19 +1,43 @@
 import { isUndefined } from 'lodash'
 
 export type PrepLevel = 1 | 2 | 3 | 4 | 5 | 6 | undefined
-export const calculatePrepLevel = (meanIntensityGroup: number | undefined): PrepLevel => {
+export const calculatePrepLevel = (
+  rawMeanIntensityGroup: number | undefined,
+  meanPrepLevel: boolean
+): PrepLevel => {
   // for now, prep level calculation assumed a fixed Fire Starts value of 0-1
-  if (isUndefined(meanIntensityGroup)) {
-    return undefined
+  if (meanPrepLevel && !isUndefined(rawMeanIntensityGroup)) {
+    const meanPrepLevelNumber = Math.round(rawMeanIntensityGroup)
+    switch (meanPrepLevelNumber) {
```

Comment:
Unnecessary switch statement, can just return the `meanPrepLevelNumber`.

Reply:
When I try and just return the rounded number I am told that number is not assignable to PrepLevel

Reply:
You can remove the `type PrepLevel` since it's not used anywhere else and just change the function signature to return `number | undefined`

Reply:
I agree with Conor's suggestion